client/public/index.html

```diff
@@ -7,7 +7,7 @@
     <!-- <link href='style.css' rel='stylesheet' type='text/css'> -->
   </head>
   <body>
-    <div id='description'></div>
-    <script src="./bundle.js"></script>
+    <div id="description"></div>
```

Comment:
Any reason for double quotes from single quotes?

Reply:
in the beginning i was using a method to pull my index.html using fs.fileRead and replace parts of the string with my root components for rendering.  to use string.replace() it required that all the single and double quotes be uniform, but I later changed this method later to the current setup.  overall: it doesn't matter much anymore lol, but beware of it if you use the string.replace() method.